### PR TITLE
wezterm: add licenses, install extra files, and update to a revision of 20210314

### PR DESCRIPTION
**Pull request by steveej**
###### Motivation for this change
* Missing licenses
* New version

Follows up on https://github.com/NixOS/nixpkgs/pull/104704 

/cc @teto 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: lib/licenses.nix [hunk 1]
```diff
@@ -55,6 +55,11 @@ lib.mapAttrs (n: v: v // { shortName = n; }) {
     free = false;
   };
 
+  angle = {
+    fullName = "ANGLE";
+    url = "https://github.com/google/angle/blob/master/LICENSE";
```
Comment (marsam) [inline lib/licenses.nix:60]: This license seems to be `bsd3`, wikipedia also register it as BSD 3 https://en.wikipedia.org/wiki/ANGLE_(software)

Comment (steveeJ) [reply lib/licenses.nix]: Thanks @marsam. I changed it to `bsd3`

**Comment by bew** [comment]
For your information, there's a new release of Wezterm from 2021-03-14:
https://wezfurlong.org/wezterm/changelog.html#20210314-114017-04b7cedd

File: pkgs/applications/terminal-emulators/wezterm/default.nix [hunk 2]
```diff
@@ -69,8 +71,15 @@ rustPlatform.buildRustPackage {
   installPhase = ''
     for artifact in wezterm wezterm-gui wezterm-mux-server strip-ansi-escapes; do
       patchelf --set-rpath "${lib.makeLibraryPath runtimeDeps}" $releaseDir/$artifact
-      install -D $releaseDir/$artifact -t $out/bin
+      install -Dsm755 $releaseDir/$artifact $out/bin/$artifact
     done
+
+    install -Dm644 assets/shell-integration/* -t $out/etc/profile.d
```
Comment (bew) [inline pkgs/applications/terminal-emulators/wezterm/default.nix:77]: This shell integration is optional to use wezterm, if you put it in `$out/etc/profile.d` does it make it _always_ loaded ?

**Comment by steveeJ** [comment]
Closing this in favor of https://github.com/NixOS/nixpkgs/pull/118712

**Pull request closed without merging**